addon/services/file-queue.js

```diff
@@ -37,7 +35,12 @@ export default class FileQueueService extends Service.extend(WithFiles) {
     @type {File[]}
     @default []
    */
-  files = A([]);
+  get files() {
```

Comment:
Should this still return an Ember array?

Reply:
Yeah I think it should. Good call - fixed in 2e0c082c3fc906e0f2d916d587e3f2507ec9d5f0